.github/workflows/build-test.yml

```diff
@@ -50,6 +50,16 @@ jobs:
       - name: Unit Test
         run: make test
 
+      - name: Integration Test V1
```

Comment:
Couldn't both v1 and new integration tests be executed in parallel?

Reply:
Good idea! However, I'd rather tackle that one in a follow-up PR :)